### ISSUE TITLE
New major awesome release

### ISSUE THREAD

**Issue by Niserd** [awesomeWM/awesome#726]
Hello. 
Could you please share some information on when will new major version of awesome be released? I know that the only correct answer is "when it is ready", but probably developers have their own approximations and plans.

**Comment by psychon** [comment]
A random source of information is https://github.com/awesomeWM/awesome/milestones/v3.6. @Elv13 is working on some improvements for client layouts that we should merge first and wait a while to shake out all bugs. I'm working on https://github.com/awesomeWM/awesome/issues/672 and would like to finish that first.
That's the current state, as far as I can think of right now.

Why do you want to have 3.6 released? Anything specifically in there that you want? What keeps you from using git/master builds?

I'll close this as a duplicate of https://github.com/awesomeWM/awesome/issues/553.

**Comment by Niserd** [comment]
Thank you very much for this information!

Actually, I am using awesome built from master because of Well Known Keyboard Layout Bug (something doesn't work when layout is not default, I do not remember what exactly doesn't work xD). I feel fine with my git-build, so 3.6 is not a must-have for me, but it would be fine to have official stable release.
